packages/block-editor/src/components/block-list/block-wrapper.js

```diff
@@ -233,11 +234,34 @@ const elements = [
 	'figure',
 ];
 
+const BlockSaveComponent = forwardRef(
+	( { children, tagName: TagName = 'div', ...props }, wrapper ) => {
+		return (
+			<BlockPropsFilterContext>
+				{ ( filter ) => (
```

Comment:
It's not entirely clear  to me what this filter is for

Reply:
It's a wrapper around the `blocks.getSaveContent.extraProps` filter.